tck/features/expressions/string/String8.feature

```diff
@@ -0,0 +1,214 @@
+    And having executed:
+      """
+      CREATE (:TheLabel {name: 'ABCDEF'}), (:TheLabel {name: 'AB'}),
+             (:TheLabel {name: 'abcdef'}), (:TheLabel {name: 'ab'}),
+             (:TheLabel {name: ''}), (:TheLabel),
+             (:TheLabel {name: ' Foo '}),
+             (:TheLabel {name: '\nFoo\n'}),
+             (:TheLabel {name: '\tFoo\t'})
+      """
+    When executing query:
+      """
+      MATCH (a)
+      WHERE a.name STARTS WITH '\n'
+      RETURN a.name AS name
+      """
+    Then the result should be, in any order:
+      | name      |
+      | '\nFoo\n' |
+    And no side effects
+
+  Scenario: [6] No string starts with null
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:TheLabel {name: 'ABCDEF'}), (:TheLabel {name: 'AB'}),
+             (:TheLabel {name: 'abcdef'}), (:TheLabel {name: 'ab'}),
+             (:TheLabel {name: ''}), (:TheLabel)
+      """
+    When executing query:
+      """
+      MATCH (a)
+      WHERE a.name STARTS WITH null
+      RETURN a
+      """
+    Then the result should be, in any order:
+      | a |
+    And no side effects
+
+  Scenario: [7] No string does not start with null
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:TheLabel {name: 'ABCDEF'}), (:TheLabel {name: 'AB'}),
+             (:TheLabel {name: 'abcdef'}), (:TheLabel {name: 'ab'}),
+             (:TheLabel {name: ''}), (:TheLabel)
+      """
+    When executing query:
+      """
+      MATCH (a)
+      WHERE NOT a.name STARTS WITH null
+      RETURN a
+      """
+    Then the result should be, in any order:
+      | a |
+    And no side effects
+
+  Scenario: [8] Handling non-string operands for STARTS WITH
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:TheLabel {name: 'ABCDEF'}), (:TheLabel {name: 'AB'}),
+             (:TheLabel {name: 'abcdef'}), (:TheLabel {name: 'ab'}),
+             (:TheLabel {name: ''}), (:TheLabel)
+      """
+    When executing query:
+      """
+      WITH [1, 3.14, true, [], {}, null] AS operands
+      UNWIND operands AS op1
+      UNWIND operands AS op2
+      WITH op1 STARTS WITH op2 AS v
+      RETURN v, count(*)
+      """
+    Then the result should be, in any order:
+      | v    | count(*) |
+      | null | 36       |
+    And no side effects
+
+  Scenario: [9] NOT with STARTS WITH
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (:TheLabel {name: 'ABCDEF'}), (:TheLabel {name: 'AB'}),
+             (:TheLabel {name: 'abcdef'}), (:TheLabel {name: 'ab'}),
+             (:TheLabel {name: ''}), (:TheLabel)
+      """
+    When executing query:
+      """
+      MATCH (a)
+      WHERE NOT a.name STARTS WITH 'ab'
+      RETURN a
+      """
+    Then the result should be, in any order:
+      | a                            |
+      | (:TheLabel {name: 'ABCDEF'}) |
+      | (:TheLabel {name: 'AB'})     |
+      | (:TheLabel {name: ''})       |
+    And no side effects
```

Comment:
missing new line

Reply:
Not here. But String4 missed one indeed. I have fixed that. Thx.